backend/src/lib.rs

```diff
@@ -732,10 +753,33 @@ pub extern "C" fn __quantum__rt__result_record_output(result: *mut c_void) {
                 .expect("Result with given id missing after expansion.")
         };
 
-        println!("RESULT\t{}", if b { "1" } else { "0" });
+        let val: i64 = if b { 1 } else { 0 };
+        output("RESULT", &val, tag, &mut std::io::stdout()).expect("Failed to write result output");
     });
 }
 
+#[cfg(windows)]
+const LINE_ENDING: &[u8] = b"\r\n";
+#[cfg(not(windows))]
+const LINE_ENDING: &[u8] = b"\n";
+
+fn output(
+    ty: &str,
+    val: &dyn std::fmt::Display,
+    tag: *mut c_char,
+    output: &mut impl std::io::Write,
+) -> std::io::Result<()> {
+    output.write_fmt(format_args!("OUTPUT\t{ty}\t{val}"))?;
```

Comment:
I didn't realize records are now prepended with `OUTPUT` unconditionally. Is that from the base profile spec?

Reply:
Yes, it will be.